src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import Scroll from 'react-scroll';
+
+import { focusElement } from '../../../../platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
```

Comment:
Since this func is only used once, no need to create its own const.  You can just move the logic to componentDidMount

Reply:
> Since this func is only used once, no need to create its own const. You can just move the logic to componentDidMount

At this present time, we're not using this file.  Our design team will likely have a need for this page at a later date so we can refactor at that point.